### PR TITLE
added no longer maintained / deprecated notice

### PR DESCRIPTION
**Pull request by lkisoslive**

### CODE REVIEW

File: pages/Documents/ClientSideConfiguration/MessagingWindowSDK/overview.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ permalink: messaging-window-sdk-overview.html
 indicator: messaging
 ---
 
+{: .deprecated}
+No longer maintained.
```
Comment (j9t) [inline pages/Documents/ClientSideConfiguration/MessagingWindowSDK/overview.md:11]: Let’s use a full sentence here—how about “This SDK is no longer maintained.”?

**Review by j9t — CHANGES_REQUESTED** [review]

**Comment by lkisoslive** [comment]
I prefer "No longer maintained" so that we can keep it standardized and
consistent across a variety of documents. Please let me know if you would
still like me to change it. Thanks, Liat

On Mon, Sep 5, 2022 at 2:45 PM Jens Oliver Meiert ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In pages/Documents/ClientSideConfiguration/MessagingWindowSDK/overview.md
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_LivePersonInc_developer-2Dcenter_pull_1861-23discussion-5Fr962814855&d=DwMFaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=ShrVgFLLAuzR8snVbrQkmq_9bmUP0A_cY3KBwAkUQBA&m=FIu6F5O-RU1DqVRMpHbGm7leJHn8A8oULtYDNyy8j6hqi4Djiaun2VpxacXf8iNk&s=nKOQqnNabr9XA_ZFxrlGvMmKVHRy0Pp-ghg062dAIg4&e=>
> :
>
> > @@ -7,6 +7,9 @@ permalink: messaging-window-sdk-overview.html
>  indicator: messaging
>  ---
>
> +{: .deprecated}
> +No longer maintained.
>
> Let’s use a full sentence here—how about “This SDK is no longer
> maintained.”?
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_LivePersonInc_developer-2Dcenter_pull_1861-23pullrequestreview-2D1096322397&d=DwMFaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=ShrVgFLLAuzR8snVbrQkmq_9bmUP0A_cY3KBwAkUQBA&m=FIu6F5O-RU1DqVRMpHbGm7leJHn8A8oULtYDNyy8j6hqi4Djiaun2VpxacXf8iNk&s=H3bT4qQHsGBnauC-pooobVINjiyUe7UL5s_NJfihPH8&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AU3MSSGCUNEDA7NZBLHM7LDV4XMMDANCNFSM6AAAAAAQELQ3H4&d=DwMFaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=ShrVgFLLAuzR8snVbrQkmq_9bmUP0A_cY3KBwAkUQBA&m=FIu6F5O-RU1DqVRMpHbGm7leJHn8A8oULtYDNyy8j6hqi4Djiaun2VpxacXf8iNk&s=txs29NBdUAyUlG74Z99M316GrVoDb6Z2A4vubZZohGI&e=>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
This message may contain confidential and/or privileged information. 
If 
you are not the addressee or authorized to receive this on behalf of the 
addressee you must not use, copy, disclose or take action based on this 
message or any information herein. 
If you have received this message in 
error, please advise the sender immediately by reply email and delete this 
message. Thank you.

**Comment by j9t** [comment]
Yes, let’s please use sentences.

**Review by j9t — APPROVED** [review]
Thanks!

**Review by j9t — APPROVED** [review]

**Pull request merged**